Umbraco-Cloud/Getting-Started/index-v7.md

```diff
@@ -25,7 +25,7 @@ With all the success you’ll have building your site with Umbraco Cloud you’l
 
 The easiest way to get started with an Umbraco Cloud project is to [take a 14-day free trial](https://umbraco.com/?product_tour_id=103162) - the project will automatically be created for you, and you are ready to get started within a few minutes.
 
-<iframe width="800" height="450" src="https://www.youtube.com/embed/_1QfmFTO6FM?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+<!-- Video has been made private, replace or remove? <iframe width="800" height="450" src="https://www.youtube.com/embed/_1QfmFTO6FM?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe> -->
```

Comment:
This is a very good point!
We should actually remove this v7-v8 version of this article, as I cannot see any immediate difference between the two.

In any case, there is an updated version of the video on the newest version of this article. 

Thanks for raising this with us.
I'll make sure the version is removed 💪